### DISCUSSION TITLE
[Feature] Tapping FaceID / top of iPhone screen should scroll to top of list

### DISCUSSION THREAD

**Discussion by odedia** [immich-app/immich#8301] [Feature Request]
### The feature

In most iOS apps, tapping the top area of the screen in the FaceID area usually results in auto-scrolling to the top of the list being presented. This is especially important in long photos stack such as Immich.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of  #1590
